### PR TITLE
feature/108@user added the component to edit user profile image

### PR DESCRIPTION
**Pull request by thebilge**

### CODE REVIEW

File: src/client/containers/i18n/de.json [hunk 1]
```diff
@@ -130,14 +130,16 @@
     },
     "UserAdmin": {
         "Editor": {
-            "page": {
-                "title": "Benutzereditor"
+            "titles": {
+                "page": "Benutzereditor",
+                "image": "Edit profile image of user {userId}"
```
Comment (kwierchris) [inline src/client/containers/i18n/de.json]: -> Profilbild von Benutzer {userId} ändern

File: src/client/containers/i18n/de.json [hunk 2]
```diff
@@ -130,14 +130,16 @@
...
             },
             "back": "Zurück",
             "tabs": {
                 "profile": "Profil",
                 "roles": "Rollen",
-                "password": "Passwort"
+                "password": "Passwort",
+                "image": "Image"
```
Comment (kwierchris) [inline src/client/containers/i18n/de.json]: -> Bild

File: src/client/containers/i18n/fi.json [hunk 3]
```diff
@@ -125,13 +125,15 @@
     },
     "UserAdmin": {
         "Editor": {
-            "page": {
-                "title": "Käyttäjän editori"
+            "titles": {
+                "page": "Käyttäjän editori",
+                "image": "Edit profile image of user {userId}"
```
Comment (kwierchris) [inline src/client/containers/i18n/fi.json]: Remove that. We can't do these translations and will use the english auto-fallback.

File: src/client/containers/i18n/fi.json [hunk 4]
```diff
@@ -125,13 +125,15 @@
...
             },
             "back": "Takaisin",
             "tabs": {
                 "profile": "Profiili",
-                "roles": "roolit"
+                "roles": "roolit",
+                "image": "Image"
```
Comment (kwierchris) [inline src/client/containers/i18n/fi.json]: Remove that. We can't do these translations and will use the english auto-fallback.

File: src/client/containers/i18n/sv.json [hunk 5]
```diff
@@ -127,13 +127,15 @@
     },
     "UserAdmin": {
         "Editor": {
-            "page": {
-                "title": "Redigarerare"
+            "titles": {
+                "page": "Redigarerare",
+                "image": "Edit profile image of user {userId}"
```
Comment (kwierchris) [inline src/client/containers/i18n/sv.json]: Remove that. We can't do these translations and will use the english auto-fallback.

File: src/client/containers/i18n/sv.json [hunk 6]
```diff
@@ -127,13 +127,15 @@
...
             },
             "back": "Tillbaka",
             "tabs": {
                 "profile": "Profil",
-                "roles": "Befattning"
+                "roles": "Befattning",
+                "image": "Image"
```
Comment (kwierchris) [inline src/client/containers/i18n/sv.json]: Remove that. We can't do these translations and will use the english auto-fallback.

File: src/client/containers/i18n/de.json (hunk 1 above)
Comment (thebilge) [reply src/client/containers/i18n/de.json]: ✓

**Pull request merged**